torch/_inductor/simple_fsdp/utils.py

```diff
@@ -130,3 +135,166 @@ def get_node_type(node: "scheduler.BaseSchedulerNode") -> NodeType:
             return NodeType.COMPUTE
 
     return _get_ir_node_type(node.node)
+
+
+def _get_benchmark_runtime(node) -> List[float]:
+    """
+    Returns estimated op runtime in nanoseconds (ns)
+    """
+    # Communication kernel benchmark
+    if is_collective(node.node):
+        # communication time for AG & RS
+        comm_time = estimate_nccl_collective_runtime(node.node)
```

Comment:
two problems:
1. it is never used, please remove. E.g. you can rename this field to be `compute_time` and set it to 0 for AG and RS
2. If it's used, we need to have the same `time_multiplier` as the bucketed one